core/admin.py

```diff
@@ -0,0 +1,14 @@
+"""Admin Panel URL Configuration for Core App
+
+This file contains all the URLs of the admin panel. This file is used to
+connect the URLs to the views.
+"""
+
+from django.contrib import admin
+from .models import Product, Category, Order, Cart
```

Comment:
For practice, I'd suggest to define your own Admin class in which you can customize/define `list_display`, `search_fields`, `list_filter`. You can look for this in django documentation.